packages/integrations/lit/README.md

```diff
@@ -141,6 +141,14 @@ export default defineConfig({
 
 The correct order might be different depending on the underlying cause of the problem. This is not guaranteed to fix every issue however, and some libraries cannot be used if you are using the Lit integration because of this.
 
+### Strict package managers
+
+When using a [strict package manager](https://pnpm.io/pnpm-vs-npm#npms-flat-tree) like `pnpm`, you may get an error like `ReferenceError: module is not defined` when running your site. To fix this, you can hoist Lit dependencies with an `.npmrc` file:
+
+```ini title=".npmrc"
+public-hoist-pattern[]=*lit* 
+```
```

Comment:
```suggestion
```ini 
title=".npmrc"
public-hoist-pattern[]=*lit* 
```
```

Reply:
Sorry, just seeing the notification now!

This format above works for docs, yes. And it's what we prefer if the code is *only* displayed in docs. 

But, if it messes up the GitHub README, then docs should *also* be able to parse:
````
```lang
// filename.here

code snippet
```
````
So that's a reasonable backup, too!

Reply:
Cool! I think because the paragraph pre-mentioned that it is a `.npmrc` file, it being omitted in GitHub readme would be fine. And the extra addition for rendered docs is a plus, so I'll stick with the change for now.